.circleci/config.yml

```diff
@@ -144,6 +135,7 @@ workflows:
 
       - remove_from_registry:
           requires:
+            - test
             - minikube_local_cluster_tests
```

Comment:
we remove the image from registry after all integration tests are completed (test use or may use it)